plugins/gateway/kong/plugin.go

```diff
@@ -0,0 +1,179 @@
+// Copyright © 2021 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"github.com/Kong/go-pdk"
+	"github.com/Kong/go-pdk/server"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	"strconv"
+	"strings"
+
+	"github.com/apiclarity/apiclarity/plugins/api/client/client"
+	"github.com/apiclarity/apiclarity/plugins/api/client/client/operations"
+	"github.com/apiclarity/apiclarity/plugins/api/client/models"
+)
+
+type Config struct {
+	apiClient *client.APIClarityPluginsTelemetriesAPI
+}
+
+// nolint: deadcode
+func New() interface{} {
+	cfg := client.DefaultTransportConfig()
+	transport := httptransport.New("apiclarity.apiclarity:9000", "/api", cfg.Schemes)
+	apiClient := client.New(transport, strfmt.Default)
+	return &Config{
+		apiClient: apiClient,
+	}
+}
+
+func (conf Config) Response(kong *pdk.PDK) {
+	telemetry, err := createTelemetry(kong)
+	if err != nil {
+		_ = kong.Log.Err(fmt.Sprintf("Failed to create telemetry. %v", err))
```

Comment:
need to be consistent with the error prints - I will use **:** and not **.** before the error print with "Failed to ...**:** %v"

Reply:
I meant in all prints in the file and not only here.